src/nvim/api/command.c

```diff
@@ -377,68 +382,96 @@ String nvim_cmd(uint64_t channel_id, Dict(cmd) *cmd, Dict(cmd_opts) *opts, Arena
     ea.argt = get_cmd_argt(ea.cmdidx);
   }
 
+  bool count_from_first_arg = false;
```

Comment:
a hint comment will be helpful here to show an example of what this means
```suggestion
  bool count_from_first_arg = false;  // "..."
```